.github/workflows/build.yaml

```diff
@@ -48,11 +68,11 @@ jobs:
     name: "python=${{ matrix.python-version }} os=${{ matrix.os }}"
     runs-on: ${{ matrix.os }}
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-python@v2
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
       - name: install tox
-        run: python -m pip install -U tox
+        run: pip install tox
       - name: test
-        run: python -m tox -e py
+        run: tox -e py
```

Comment:
These invocations don't matter. In the context of GitHub Actions, `python -m tox` and `tox` are the same.
However, it's nicer netiquette (IMO) to leave things as they are unless there's a need to change them. I'd rather _not_ have to rethink the invocation of tox in a PR which isn't explicitly about or impacted by that.